ReactWindows/ReactNative/Views/TextInput/ReactClearButtonModeHelper.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace ReactNative.Views.TextInput
+{
+    /// <summary>
+    /// Modes for the visibility state of the clear button in a ReactTextBox.
+    /// </summary>
+    internal enum ClearButtonModeType
```

Comment:
>ClearButtonModeType [](start = 18, length = 19)

Looks like you'll have to move this to the shared project, ReactNative.Shared.

Reply:
Is that why the continuous-integration test is failing with "ClearButtonModeType could not be found"?

Reply:
Yep.

Reply:
All done